web_external/ImagesGallery/PagingPane.js

```diff
@@ -6,7 +6,11 @@ isic.views.ImagesViewSubViews.PagingPane = Backbone.View.extend({
         'click #isic-images-seekFirst': 'seekFirst',
         'click #isic-images-seekPrev': 'seekPrev',
         'click #isic-images-seekNext': 'seekNext',
-        'click #isic-images-seekLast': 'seekLast'
+        'click #isic-images-seekLast': 'seekLast',
+        'click #isic-images-download-zip': function () {
+            window.location.assign(girder.apiRoot + '/image/download?filter=' +
+                JSON.stringify(this.model.getFilterAstTree()));
+        }
```

Comment:
Good catch. The bug here is really the failure to apply the rule from https://github.com/ImageMarkup/isic-archive/pull/352 to this endpoint. Fixed now.